tasks/components/Pagination.tsx

```diff
@@ -0,0 +1,64 @@
+import { tw, tx } from '@helpwave/common/twind/index'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import ChevronLast from '@helpwave/common/icons/ChevronLast'
+import ChevronLeft from '@helpwave/common/icons/ChevronLeft'
+import ChevronFirst from '@helpwave/common/icons/ChevronFirst'
+import ChevronRight from '@helpwave/common/icons/ChevronRight'
+
+type PageIndicatorTranslation = {
```

Comment:
update the name here as well (there are still a few other instances using `pageIndicator`)